src/shell/Command.ts

```diff
@@ -78,6 +78,13 @@ const executors: Dictionary<(i: Job, a: string[]) => void> = {
             throw `Don't know what to do with ${args.length} arguments.`;
         }
     },
+    show: (job: Job, args: string[]): void => {
+        let imgs: string[] = [];
+        args.forEach(argument => {
+            imgs.push(resolveDirectory(job.environment.pwd, argument).slice(0, -1));
```

Comment:
There is a `resolveFile` function to avoid a slash at the end. Also, `const images = args.map(...)` would be better.